skytemple_files/common/ppmdu_config/pmdsky_debug/test/migration_test.py

```diff
@@ -0,0 +1,244 @@
+#  Copyright 2020-2022 Capypara and the SkyTemple Contributors
+#
+#  This file is part of SkyTemple.
+#
+#  SkyTemple is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  SkyTemple is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with SkyTemple.  If not, see <https://www.gnu.org/licenses/>.
+import unittest
+
+
+# First tuple entry:   Has end?
+# Second tuple entry:  type == Pmd2BinarySymbolType.FUNCTION
+from parameterized import parameterized
+
+from skytemple_files.common.ppmdu_config.data import Pmd2GameEdition, GAME_VERSION_EOS, GAME_REGION_EU, GAME_REGION_US
+from skytemple_files.common.ppmdu_config.pmdsky_debug.data import SymbolHasNoEndError, Pmd2BinarySymbolType
+from skytemple_files.common.ppmdu_config.pmdsky_debug.loader import load_binaries
+
+OLD_ENTRIES = {
+    'arm9.bin': {'CartRemovedImgData': (True, False),
+                 'CompressedIqGroupsSkills': (True, False),
+                 'DebugPrint': (False, True),
+                 'DebugPrint0': (False, True),
+                 'DebugPrint2': (False, True),
+                 'DebugSpecialEpisodeType': (False, False),
+                 'DefaultHeroId': (True, False),
+                 'DefaultPartnerId': (True, False),
+                 #'DungeonData': (False, False),
```

Comment:
I'm guessing these commented-out entries were never used by SkyTemple? Most of them (maybe not all though) should still be in `pmdsky-debug` under different names I think.

Reply:
Yeah, they either weren't used or I updated all references to them to the new name manually for one reason or another.